shell/platform/darwin/ios/framework/Source/UIViewController+FlutterScreenAndSceneIfLoaded.mm

```diff
@@ -0,0 +1,30 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/ios/framework/Source/UIViewController+FlutterScreenAndSceneIfLoaded.h"
+
+#include "flutter/fml/logging.h"
+
+@implementation UIViewController (FlutterScreenAndSceneIfLoaded)
+
+- (UIWindowScene*)flutterWindowSceneIfViewLoaded {
```

Comment:
The name is long already and I'm not sure the value of adding `flutter`. Can we discuss leaving the name `windowSceneIfViewLoaded`? I think it would result in less change to FlutterViewController as well.

Reply:
Fun story about ObjC is that it has no namespace support. So if someone created a category with the same function name, and even if they import their own category header, it's indeterministic which function of the 2 categories will be called at run-time. (That's right, the category name is not a namespace!)

So to deal with this, the rule of thumb for defining categories of system types (e.g. UIViewController) is to avoid naming conflict of functions in the first place, by adding a prefix. 

I agree the name is a bit long. I was going to use flt_, but looks like flutter prefix is the convention.